src/randgraphs.jl

```diff
@@ -96,11 +96,10 @@ end
 
 function _suitable(edges::Set{Edge}, potential_edges::Dict{Int, Int})
     isempty(potential_edges) && return true
-    for (s1, s2) in combinations(collect(keys(potential_edges)), 2)
-        if (s1 > s2)
-            s1, s2 = s2, s1
-        end
-        ∉(Edge(s1, s2), edges) && return true
+    list = keys(potential_edges)
+    for s1 in list, s2 in list
```

Comment:
keys returns an iterator, so one should do
```julia
list=sort(collect(keys(potential_edges)))
```
So we loose some time in allocating, and we gain some in doing less cycles (half) and avoiding one branching  `&&`
I really don't know what is the optimal thing to do here. I don't think this is really relevant so let's choose one way or the other, both are fine with me